docs/04-operation-guides/troubleshooting/evnt-02-jetstream-troubleshooting.md

```diff
@@ -0,0 +1,50 @@
+---
+title: JetStream backend troubleshooting
+---
+
+## Symptom
+
+Events were not received by the consumers.
+
+## Remedy
+
+1. Follow the basic troubleshooting steps as mentioned in [Eventing Troubleshooting](./evnt-01-eventing-troubleshooting.md) guide.
+
+2. Use the [nats CLI](https://github.com/nats-io/natscli) to check if the stream was created:
+
+    1. Port forward the Kyma Eventing NATS Service to localhost. Use port `4222`. Run:
+        ```bash
+        kubectl -n kyma-system port-forward svc/eventing-nats 4222
+        ```
+    2. Use the [nats CLI](https://github.com/nats-io/natscli) to list the streams:
+       ```bash
+       nats stream ls
+       ```
+    
+        ```bash
+        ╭────────────────────────────────────────────────────────────────────────────╮
+        │                                  Streams                                   │
+        ├──────┬─────────────┬─────────────────────┬──────────┬───────┬──────────────┤
+        │ Name │ Description │ Created             │ Messages │ Size  │ Last Message │
+        ├──────┼─────────────┼─────────────────────┼──────────┼───────┼──────────────┤
+        │ sap  │             │ 2022-05-03 00:00:00 │ 0        │ 318 B │ 5.80s        │
+        ╰──────┴─────────────┴─────────────────────┴──────────┴───────┴──────────────╯
+        ```       
+
+    3. If the stream exists, check the timestamp of the `Last Message` that the stream received. A recent timestamp would mean that the event was published correctly.
+   
+    4. Check if the consumers were created and have the expected configurations.
+       ```bash
+       nats consumer info
+       ```
```

Comment:
Why don't we show the output of the command here ? We do it for `nats stream ls`. I think it can help the user because the user might not know what `consumers were created and have the expected configurations` means.

Reply:
In my opinion, I think we don't need to show it here since the user will have completely different consumer names, and values and our output will not benefit the user. It's just here to tell the user that this command is also present and he can use it just like the `nats stream ls` command